src/isIsogram.test.js

```diff
@@ -6,4 +6,47 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`should return true if users nothing enters`, () => {
+    expect(isIsogram(''))
+      .toBeTruthy();
+  });
+
+  it(`should return true if users enters isogram`, () => {
+    expect(isIsogram('beach'))
+      .toBeTruthy();
+  });
+
+  it(`should return false if users enters repeating letters`, () => {
```

Comment:
Rename the test description to 'should return false if a word with repeating letters is entered' for better clarity 📝

Reply:
Fixed.